package.json

```
@@ -210,6 +210,7 @@
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.6",
     "copy-to-clipboard": "^3.1.0",
+    "core-js": "3",
```

This looks rather odd
```suggestion
    "core-js": "^3.0.0",
```